src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -2989,16 +3037,16 @@ internal abstract partial class MemberReferenceExpression : Operation, IMemberRe
         /// <summary>
         /// Referenced member.
         /// </summary>
-        public ISymbol Member { get; }
+        public virtual ISymbol Member { get; }
     }
 
     /// <summary>
     /// Represents a reference to a method other than as the target of an invocation.
     /// </summary>
     internal abstract partial class BaseMethodBindingExpression : MemberReferenceExpression, IMethodBindingExpression
     {
-        public BaseMethodBindingExpression(IMethodSymbol method, bool isVirtual, ISymbol member, SemanticModel semanticModel, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue, bool isImplicit) :
-            base(member, OperationKind.MethodBindingExpression, semanticModel, syntax, type, constantValue, isImplicit)
+        public BaseMethodBindingExpression(IMethodSymbol method, bool isVirtual, SemanticModel semanticModel, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue, bool isImplicit) :
+            base(null, OperationKind.MethodBindingExpression, semanticModel, syntax, type, constantValue, isImplicit)
```

Comment:
>null [](start = 17, length = 4)

Passing null here feels wrong. It looks like that will result in IMemberReferenceExpression.Symbol to be null. I think we should pass ```method``` instead.

Reply:
> It looks like that will result in IMemberReferenceExpression.Member to be null. 

I see that ```MemberReferenceExpression.Member``` is overridden below to prevent that. Still, I think it is better to pass ```method``` and, instead of making the ```Member``` property virtual, implement ```Method``` like ```public IMethodSymbol Method => (IMethodSymbol)Member;```. This way we won't have a need for a separate field to store the method.

---
In reply to: [140099464](https://github.com/dotnet/roslyn/pull/22056#discussion_r140099464) [](ancestors = 140099464)